### PR TITLE
Test Cases Added

### PR DESCRIPTION
**Pull request by alecbuetow**
Can we keep this pull request open for a few hours? I think Github is registering the merge as one commit on the upstream repo but still tracking every individual commit in my forked repo, which is forcing me to re-sync for every merge. I want to check if committing directly to the PR resolves this

### CODE REVIEW

**Comment by talgalili** [comment]
Sure

**Comment by talgalili** [comment]
I suggest to not work on `unloadNamespace` and the code there. I'd suggest keep it to the end of the work.
It's best to focus on testing core functions.

**Comment by alecbuetow** [comment]
Sounds good, done playing with it

**Comment by talgalili** [comment]
Cool. Please let me know once I can review this PR

File: R/ggdend.R
Comment (talgalili) [inline R/ggdend.R:1]: Why did you make this change? i believe it will break things.

**Review by talgalili — CHANGES_REQUESTED** [review]

File: R/ggdend.R
Comment (alecbuetow) [reply R/ggdend.R:1]: I was getting the following warning, which was resolved when `size` was switched to `linewidth`. You're right though, size = 1 and linewidth = 1 won't set the line to the same width. Better to go with the old version

Warning (test-ggdend.R:151:4): ggplot doesn't have warnings for dendrograms
Using `size` aesthetic for lines was deprecated in ggplot2
3.4.0.
ℹ Please use `linewidth` instead.
Backtrace:
    ▆
 1. ├─ggplot2::ggplot(as.dendrogram(hclust(dist(mtcars)))) at test-ggdend.R:151:4
 2. └─dendextend:::ggplot.dendrogram(as.dendrogram(hclust(dist(mtcars))))
 3.  ├─ggplot2::ggplot(as.ggdend(data), ...) at dendextend/R/ggdend.R:635:3
 4.  └─dendextend:::ggplot.ggdend(as.ggdend(data), ...)
 5.    └─ggplot2::geom_segment(...) at dendextend/R/ggdend.R:537:6
 6.      └─ggplot2::layer(...)
 7.        └─ggplot2:::deprecate_soft0(...)

**Comment by talgalili** [comment]
Thanks, merged.

Keep them coming :)


Regarding the size vs linewidth, maybe open an issue on this so we won't forget to find a solution for this in the future?
Thanks!

**Pull request merged**